### ISSUE TITLE
Motion to Vacate | Serialize appeal stream information

### ISSUE THREAD

**Issue by nanotone** [department-of-veterans-affairs/caseflow#13333]
## Description

In order to support UI flows for MTV, the frontend needs serialized appeal objects to include the appeal's stream type and vacate type.

## Acceptance criteria

- [ ] Appeal serializer includes `type`
- [ ] Appeal serializer includes `vacate_type` when stream type is `Vacate`

## Background/context

Appeal streams are separate Appeal objects that share the same docket number but different stream types, and indicate work that can be done in parallel on a case. For a Motions to Vacate flow, the stream type is `Vacate`, and the vacate type for a granted motion indicates whether it is a straight vacate, vacate and readjudication, or vacate and de novo.

Because much of the MTV flow is shared with that of a normal Appeal, both in terms of frontend UI and backend data modeling, we want to expose stream type and vacate type information in the Appeal serializer so that the frontend can display the correct UI.

## Technical notes

The Appeal serializer lives in `app/serializers/intake/appeal_serializer.rb`. The stream type is available via the `Appeal#type` method.

The vacate type, which should only be present when the stream type is `Vacate`, is a bit more complicated to retrieve: it exists as `PostDecisionMotion#vacate_type` via the post-decision motion whose task is a `JudgeAddressMotionToVacateTask` on the appeal's task tree.

**Comment by jcq** [comment]
We will also need to provide the `vacated_issues` for the `PostDecisionMotion` associated with the task/appeal to the frontend. Does it make sense to add that here?

If this seems out of scope, we could always create another ticket for it.

**Context**
https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/models/post_decision_motion.rb#L24

**Comment by nanotone** [comment]
@jcq Good catch! Yeah, it could totally go here -- would it be okay to include as an extra field on the serialized appeal?

**Comment by jcq** [comment]
After talking to @leikkisa, my question might actually be moot. While obtaining them from the *original* appeal is a bit complicated (and what I was thinking with my first question), it's trivial to get them from the *new* appeal stream — they will just be *the* decision issues for that appeal.
